runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -1925,20 +1925,21 @@ J9::Z::CodeGenerator::markStoreAsAnAccumulator(TR::Node *node)
    }
 
 
-/// If true, this node's operation might overwrite an accumulator by evaluating one child before loading
-/// the value from another, if we choose to accumulate. (Accumulation may still be safe, but we'll need
-/// to investigate all child nodes to be sure).
-/// eg.
-//
-/// pdstore "a"
-///   pdsub
-///     pdconst
-///     zd2pd
-///       zdload "a"
-///
-/// Accumulating to "a" is incorrect here because the pdconst will get evaluated into "a" before the
-/// zdload is evaluated, so when we encounter the pdsub, we need to check all children.
-///
+/**
+  * If true, this node's operation might overwrite an accumulator by evaluating one child before loading
+  * the value from another, if we choose to accumulate. (Accumulation may still be safe, but we'll need
+  * to investigate all child nodes to be sure).
+  * eg.
+  *
+  * pdstore "a"
+  *   pdsub
+  *     pdconst
+  *     zd2pd
+  *       zdload "a"
+  *
+  * Accumulating to "a" is incorrect here because the pdconst will get evaluated into "a" before the
+  * zdload is evaluated, so when we encounter the pdsub, we need to check all children.
+  */
```

Comment:
Please remove one space before the asterisk on each of these lines so that they line up with the first asterisk in `/**`